app/src/main/java/de/westnordost/streetcomplete/data/osmnotes/NotesDownloader.kt

```diff
@@ -44,6 +40,6 @@ private suspend fun NotesApi.getAll(
     hideClosedNoteAfter: Int
 ): List<Note> = withContext(Dispatchers.IO) {
     val notes = ArrayList<Note>()
-    getAll(bbox, notes::add, limit, hideClosedNoteAfter)
+    getAll(bbox.toBoundingBox(), notes::add, limit, hideClosedNoteAfter)
```

Comment:
this looks wrong. BoundingBox is already a BoundingBox, why does it have a method called toBoundingBox ?

In any case, notesApi should accept a de.westnordost.streetcomplete.data.osmnotes.toBoundingBox .